providers/widget/publish/main.js

```diff
@@ -18,7 +18,7 @@ try {
 	
     utils.execStep(
 		[
-			`npx npm-snapshot ${buildNumber}`,
+			`npx github:dominguezcelada/npm-snapshot ${buildNumber} snapshot`,
```

Comment:
I can merge it to my master if you prefer

(also I opened a PR to original repo just in case 😇 )

Reply:
yep, lets merge it

Reply:
```suggestion
			`npx dominguezcelada/npm-snapshot ${buildNumber} snapshot`,
```

💅🏽